omod/src/main/java/org/openmrs/module/dataintegrity/web/rest/controller/DataIntegrityRestSubController.java

```diff
@@ -0,0 +1,18 @@
+package org.openmrs.module.dataintegrity.web.rest.controller;
```

Comment:
This class shouldn't be needed.... Try removing it and see if it fixes the issue.